app/models/user.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  attr_accessor :remember_token
+  validates :name, presence: true, uniqueness: { case_sensitive: false }
+  has_many :events
```

Comment:
the User that creates and event should be referred to as the **creator**  foreign key should be referenced here